.env.example

```
@@ -16,32 +25,32 @@ USER_AGENT=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML,
 CRAWLER_MAX_RETRIES=3                                          # Maximum retry attempts for downloads
 CRAWLER_HEADLESS=true                                          # Run browser in headless mode (true/false)
 
-# Processing Configuration
-OUTPUT_DIR=./output                                            # Directory for processed output files
-
 # Working Directory Configuration
 WORKING_DIR=./.govdoc                                          # Base directory for all app outputs (default: ./.govdoc in project root)
                                                                # Creates: ./.govdoc/crawler/, ./.govdoc/doc-scanner/, ./.govdoc/cli/
                                                                # Use absolute path or ~ for home directory if desired
 
-# Logging Configuration
-LOG_LEVEL=error                                                # Logging level: debug, info, warn, error
-
-# OpenSearch Integration (Optional)
+# OpenSearch Integration
 # Configure based on your chosen environment (Delete the one not in use):
-OPENSEARCH_PUSH=true
+OPENSEARCH_PUSH=true                                           # Enable pushing data to OpenSearch
```

False should be the default here
```suggestion
OPENSEARCH_PUSH=false                                           # Enable pushing data to OpenSearch
```